purchase_last_price_info/migrations/11.0.1.0.0/post-migration.py

```diff
@@ -0,0 +1,14 @@
+# Copyright 2019 Eficent Business and IT Consulting Services S.L.
+#   (http://www.eficent.com)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+from odoo import api, SUPERUSER_ID
+
+
+def migrate(cr, version):
+    if not version:
+        return
+    with api.Environment.manage():
```

Comment:
Don't! It's better to reuse the post_init_hook to avoid duplicated code.

I think you should do:

```python
from openupgradelib import openupgrade
from odoo.addons.purchase_last_price_info import set_last_price_info


@openupgrade.migrate()
def migrate(env, version):
    set_last_price_info(env.cr, env.registry)
```